server/bin/pbench-restore-and-unpack-tarballs.py

```diff
@@ -103,23 +122,39 @@ def main(options):
         print(f"{_NAME_}: {e}", file=sys.stderr)
         return 2
 
-    archive = config.ARCHIVE
-    archive_p = Path(archive).resolve(strict=True)
+    try:
+        archive_p = Path(config.ARCHIVE).resolve(strict=True)
+    except Exception as e:
```

Comment:
I think you should be catching `FileNotFoundError` here, rather than `Exception` -- `resolve()` _can_ raise other exceptions, and they indicate things other than "file does not exist".  (I'm not trying to say that you need to catch additional cases, here; rather, I think that we shouldn't muddy the waters if we hit a weird case.)

Reply:
Ditto line 146.